### ISSUE TITLE
"center" of VR view in Unity is on left side of 360 vide

### ISSUE THREAD

**Issue by rms80** [facebook/360-Capture-SDK#8]
I am trying to use EncodePackage in my own Unity project. It works great (thanks!!), however in the rendered video the thing that I see in the "middle" of my VR view is on the left side. So, when you start it in any VR video player, you have to "turn left" to see anything. 

I would like the ground-plane object to be where the red X is in the screenshot.

The EncoderObject prefab is pointed at my scene, as one would expect. Is there some kind of global rotation being applied to the captured video? Or, can I apply such a rotation?

![image](https://cloud.githubusercontent.com/assets/2845837/26175909/4e04bb92-3b22-11e7-95ba-dc8e5b14a46c.png)

**Comment by grancia** [comment]
You can rotate with following instruction.

1. Set "includeCameraRotation" variable in SurroundCapture.cs to true
2. Adjust "Y" value in the Transform of EncoderObject.

**Comment by rms80** [comment]
includeCameraRotation=true had no effect (perhaps because I am using Equirect and not CubeMap mode?). I tried changing transforms on EncoderObject and the child Camera and they also didn't do anything.

However I discovered that SurroundCapture.sphereOffset.x will rotate around the up axis, and is available from the Unity editor. Setting it to -0.25 rotated right by 90 degrees. So, it's not a degree/radian value, is it maybe multiplied by 2*pi ?

![image](https://cloud.githubusercontent.com/assets/2845837/26207005/564e0622-3bb4-11e7-8132-d634a1a5a291.png)
